tests/run_tests.sh

```diff
@@ -25,6 +25,12 @@ function print_graphics_data () {
 }
 
 
+
+function detect_known_issues () {
+	cat "$1" | grep "SDL message: \"No available video device\""
```

Comment:
Let's make this easily extendable, for e.g. 0 to N different issues, so a PR to add/remove one is a single line.

Reply:
Agreed. I moved the issues list to a separate file.